utilz/map_update.rb

```diff
@@ -78,7 +80,7 @@ def self.geocode_projects
       project[:yml]["Pcrds"] = pCrds
       project[:yml]["Hcrds"] = hCrds
 
-      File.open(file,"w"){|f| f.write("#{project[:yml].to_yaml}---\n\n#{project[:description]}")}
+      File.open(file,"w"){|f| f.write("#{project[:yml].to_yaml}---#{project[:description]}")}
```

Comment:
added because the '\n\n' kept adding two extra lines... unnecessarily it turns out

Reply:
hmm, i'd think atleast one would be needed? can you share the output of some project md file?

Reply:
---
layout: project
volume: 2011
image: GIN_BLOSSOMS_AND_BROCCOLI.jpg
photo_credit: NF Chase
title: GIN BLOSSOMS AND BROCCOLI BOUTONNIÈRES
first_performed: March  2, 2011
place: Meridian Gallery, San Francisco, CA
times_performed: performed twice in 2011
contributor: NF CHASE
collaborators:
- Jill Heinke
home: Oregon
links:
- http://nfchase.net
contact: nchase.arts@gmail.com
footnote: 
tags:
- audio
- choice
- composition
- computer
- document
- grammar
- improvisation
- language
- meaning
- music
- music rhythm
- notation
- process
- real time
- repetition
- score
- sound
- spontaneity
- time
- tradition
- visceral
- weave
- word
- working
pages: '084-085'
Pcrds:
- 37.8009466,
- "-122.4513899"
Hcrds:
- 43.8041334,
- "-120.5542012"
---

“Gin Blossoms and Broccoli Boutonnières” is a music performance for flute mediated by laptop computer. I envisioned the flute performer as a kind of DJ, whose musical input controls, alters, and “scratches” the beat, tempo, pitch, forward- and backward-playback of pre-recorded audio material that is arbitrated by computer. In response, the computer alters percussive and wind sounds produced by the flute, and integrates them into the fabric of the pre-recorded materials. The result is a complex weave of melodic rhythm and sound bites. The musical form is un-fixed and can be spontaneously arranged into a variety of structural possibilities. The result is paradoxical: “Gin Blossoms and Broccoli Boutonnières” is a repeatable musical act that will have unique results each time it is performed.

In this way, the performance can address issues of traditional, consistently repeatable musical performance 1) by challenging the performer/s to execute composed material in a manner that is musically unfamiliar, and 2) by engaging the performer/s in a real-time/spontaneous, visceral decision-making process that is specific to each performance.

Working with traditional music notation was problematic: the printed music for “Gin Blossoms and Broccoli Boutonnières” should be useable as a guide for future performances, as would be a traditional score-yet it should offer an open-endedness that facilitates uniqueness in each performance-a feature inherent to improvisation, which follows no printed score.

To tackle this problem, I abandoned the notion of the musical score as document of fixed, repeatable ideas, and developed a graphically oriented musical language that provides both consistency and room for spontaneity-free choice.

The core material for the composition is a single musical phrase supported by a series of secondary phrases. In the way that words in a sentence use grammatical modifiers and can be arranged in different syntax for alternate meanings, I broke down these musical phrases into musical “phonemes” that can, themselves, be rearranged to create new musical ideas. In this way, the core material of the music remains constant, while the musical form is unfixed and can be spontaneously created-and endlessly re-created-into any variety of structural possibilities, contributing to an enervated and vibrant musical result that is new in every performance.

Reply:
oh thanks! 

...any thoughts on the type format for the coordinates? like i wonder if just a single string would be easier than having an array of number and a string? it's unfortunate that yaml needs to quote negative numbers... 

something like: `Pcrds: "37.8009466 -122.4513899"`?

Reply:
The negative quoting is def weird - will implement that on next commit. Also Pcrds and Hcrds... we can probably think of better names... but they'd add more to the JSON files, is it that big of a different to go with place_coords or home_coords, or p_loc and h_loc?